velox/core/PlanNode.h

```diff
@@ -866,15 +900,28 @@ class LocalPartitionNode : public PlanNode {
     }
   }
 
+  /// TODO Remove once Prestissimo is updated.
+#ifdef VELOX_ENABLE_BACKWARD_COMPATIBILITY
```

Comment:
Do we plan to rename PartitionFunctionSpec to PartitionFunctionFactory after we remove this?